### PR TITLE
Recognition performance analytics

### PR DESCRIPTION
**Pull request by sandera0606**
Updated MainScene UI. Added AnalyticsDisplay script which currently only displays the inference output & the confidence threshold.

Added UWRHandPoseDetection script. Allows threshold confidence level for inference to be specified in the unity inspector.

### CODE REVIEW

**Comment by JustinLin905** [comment]
Can you summarize any changes that you might have made to the MainScene? Just want to understand so we can avoid conflicts

**Comment by sandera0606** [comment]
Hi! I added the script "UWRL Hand Pose Detection" to the gameobject Scripts and I added some empty gameobjects and TMPro elements under the "Inference Status UI" gameobject. Please let me know if theres anything else you need to know!

**Review by JustinLin905 — APPROVED** [review]

**Comment by JustinLin905** [comment]
Let's merge it! Feel free to have the honour of hitting the button Sandra haha

**Comment by sandera0606** [comment]
Wow! Honoured 💥💥
